tests/unit/backend/corpora/api_server/curator/collection/test_curation_collections.py

```diff
@@ -128,6 +128,61 @@ def test__create_collection__InvalidParameters(self):
                     self.assertIn(error, response.json["detail"])
 
 
+class TestGetCollections(BaseAuthAPITest):
+    def setUp(self):
+        super().setUp()
+        self.test_collection = dict(
+            name="collection", description="description", contact_name="john doe", contact_email="johndoe@email.com"
+        )
+
+    def test__get_collections_no_auth__OK(self):
+        res_no_auth = self.app.get("/curation/v1/collections")
+        self.assertEqual(200, res_no_auth.status_code)
+        self.assertEqual(6, len(res_no_auth.json["collections"]))
+        [self.assertEqual("PUBLIC", c["visibility"]) for c in res_no_auth.json["collections"]]
+
+    def test__get_collections_no_auth_processing_status_PENDING__OK(self):
```

Comment:
you could make this a subtest of the above test.

Reply:
I like this idea. I've seen you use this pattern and I should be using it more.